code/_globalvars/lists/misc.dm

```diff
@@ -41,6 +41,40 @@ GLOBAL_LIST_INIT(cooking_recipes, list(RECIPE_MICROWAVE = list(), RECIPE_OVEN =
 GLOBAL_LIST_INIT(cooking_ingredients, list(RECIPE_MICROWAVE = list(), RECIPE_OVEN = list(), RECIPE_GRILL = list(), RECIPE_CANDY = list()))
 GLOBAL_LIST_INIT(cooking_reagents, list(RECIPE_MICROWAVE = list(), RECIPE_OVEN = list(), RECIPE_GRILL = list(), RECIPE_CANDY = list()))
 
+GLOBAL_LIST_INIT(space_laws ,list("100, Damage to Station Assets" = "To deliberately damage the station or station property to a minor degree with malicious intent.",
+			"102, Battery" = "To use minor physical force against someone without intent to seriously injure them.",
```

Comment:
We've rejected PRs in the past for hardcoding space law into the code. I don't really see any reason this should be different. If anything, the space law book should be an iframe pointing to our wiki.

Reply:
understandable. Will it be okay if I pull the data directly from the wiki? I'm thinking of parsing the site and getting the data out of the table. Unless it's saved somewhere more readable that is.